mapreader/classify/datasets.py

```diff
@@ -600,20 +701,20 @@ def plot_sample(self, indx: int) -> None:
         """
         plt.figure(figsize=(10, 5))
         plt.subplot(1, 2, 1)
-        plt.imshow(transforms.ToPILImage()(self.__getitem__(indx)[0]))
+        plt.imshow(transforms.ToPILImage()(self.__getitem__(idx)[0]))
         plt.title("Patch", size=18)
         plt.xticks([])
         plt.yticks([])
 
         plt.subplot(1, 2, 2)
-        plt.imshow(transforms.ToPILImage()(self.__getitem__(indx)[1]))
+        plt.imshow(transforms.ToPILImage()(self.__getitem__(idx)[1]))
         plt.title("Context", size=18)
         plt.xticks([])
         plt.yticks([])
         plt.subplot(1, 2, 2)
         plt.show()
 
-    def __getitem__(self, idx: int) -> Tuple[torch.Tensor, torch.Tensor, Any]:
+    def __getitem__(self, idx: Union[int, torch.Tensor]) -> tuple:
```

Comment:
Perhaps add the type hinting in here, from the Returns section below? Makes the codebase so easily usable in VS code and other environs.